tests/garage/experiment/test_experiment_wrapper.py

```diff
@@ -0,0 +1,27 @@
+import base64
+import unittest.mock
+
+import cloudpickle
+
+from garage.experiment import SnapshotConfig
+from garage.experiment.experiment_wrapper import run_experiment
+
+
+class TestExperimentWrapper(unittest.TestCase):
+    @staticmethod
+    def method_call(snapshot_config, variant_data):
+        assert isinstance(snapshot_config, SnapshotConfig)
```

Comment:
can you assert anything more about variant_data and snapshot_config?

Reply:
Probably not as these are the only two return values.

Reply:
umm can't you assert what's inside SnapshotConfig?